packages/framework/ai-collab/src/explicit-strategy/promptGeneration.ts

```diff
@@ -82,14 +82,18 @@ export function getPlanningSystemPrompt(
 			The other agent follows this guidance: ${systemRoleContext}`
 	}`;
 
+	const editOptions = doesNodeContainArraySchema(treeNode)
+		? "inserting, modifying, removing, or moving elements in the tree"
+		: "modifying elements in the tree";
```

Comment:
I would have thought that "modifying elements in the tree" could always be there, and the other bit be the only conditional part. Also, would it help if that conditional bit read something like "as well as inserting, removing, or moving elements inside the nodes of the tree which are arrays according to the schema"?